lib/v8.js

```diff
@@ -79,3 +82,23 @@ exports.getHeapSpaceStatistics = function() {
 
   return heapSpaceStatistics;
 };
+
+exports.Serializer =
+class Serializer extends serdesBinding.Serializer {
+  constructor(options = {}) {
```

Comment:
Extremely minor nit but use of argument defaults has not benchmarked rather well yet. It's likely not something to worry about in this case, however